modules/core/module.go

```diff
@@ -217,3 +217,8 @@ func (AppModule) ProposalMsgs(simState module.SimulationState) []simtypes.Weight
 func (am AppModule) RegisterStoreDecoder(sdr simtypes.StoreDecoderRegistry) {
 	sdr[exported.StoreKey] = simulation.NewDecodeStore(*am.keeper)
 }
+
+// WeightedOperations returns the all the ibc module operations with their respective weights.
```

Comment:
Are these relevant in 0.50 @aljo242?

Reply:
yes, this is part of the [AppModuleSimulation](https://github.com/cosmos/cosmos-sdk/blob/b9db4d29aaeaf95887c5363c8050dc342ed4f1a2/types/module/simulation.go#L18) interface in 50 (was removed in 52)